ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -45,7 +53,7 @@ struct ToDoListView: View {
                     .listStyle(.plain)
                 }
                 Button( action: {
-                    viewModel.addButtonTapped()
+                    viewModel.addEditButtonTapped(mode: .createNewTask)
```

Comment:
Actually the view should not be aware of those changes. Internal interfaces have to remain the same add tapped, edit tapped, but ViewModel has to invoke private method with either mode depending on the tapped button